hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SystemSetup.java

```diff
@@ -194,6 +200,25 @@ ctx, createFileID(filesConfig.hapiPermissions(), config), bytes),
         if (autoNodeAdminKeyUpdates.tryIfPresent(adminConfig.upgradeSysFilesLoc(), systemContext)) {
             dispatch.stack().commitFullStack();
         }
+        // In 0.57 specifically, we export synthetic node updates for all non-deleted nodes in the address book,
+        // exposing their admin keys to mirror nodes to supplement the NodeAddressBook message in 0.0.112
```

Comment:
Does this then need to be removed after 0.57 so it doesn't continually send synthetic node updates in every future release? Should it be moved to a migration instead?

Reply:
Hmm @steven-sheehy not sure I understand; this is being added just in `release/0.57` not `develop`.

Reply:
Nevermind me then. I'm used to changes always going to main and cherry picked to a release branch.